### DISCUSSION TITLE
Feature Request: add support to Excel 2003 format (.xls) and via http visit

### DISCUSSION THREAD

**Discussion by l1t1** [duckdb/duckdb#9018] [Ideas]
Why do you want this feature?
because duckdb can read/write Excel 2007(.xlsx) only and can only visit local file ,it cannot visit Excel files via http/https.

**Comment by Maxxen** [comment]
You will be able to read xlsx remotely with spatial and httpfs next release!

**Reply by l1t1** [reply]
select * from st_read('http://aaa/bbb.xlsx',layer='Sheet1'); works in version 0.9

**Comment by l1t1** [comment]
select * from st_read('http://aaa/bbb.xls',layer='Sheet1') ----------------Excel 2003
Error: IO Error: GDAL Error (4): Failed to read TopoJSON data

**Comment by djouallah** [comment]
got different errors

```
import duckdb
url = "https://www.aemo.com.au/-/media/Files/Electricity/NEM/Participant_Information/NEM-Registration-and-Exemption-List.xls"
sheet_name="Generators and Scheduled Loads"
duckdb.sql(f''' INSTALL spatial; LOAD spatial;''')
duckdb.sql(f''' select * from st_read('{url}',layer='{sheet_name}') ''').show()
```

IOException: IO Error: GDAL Error (1): error setting certificate file: /etc/pki/tls/certs/ca-bundle.crt

**Comment by b-schmeling** [comment]
@djouallah 
I know this is an old discussion but did you ever resolve this error? I am receiving the same one: 
> (duckdb.duckdb.IOException) IO Error: GDAL Error (1): error setting certificate file: /etc/pki/tls/certs/ca-bundle.crt')

**Reply by Maxxen** [reply]
Do you have the `httpfs` extension installed? What version of DuckDB are you running? You can get install the latest nightly spatial version with all the recent fixes for DuckDB v0.9.2 using:

```sql
FORCE INSTALL spatial FROM 'http://nightly-extensions.duckdb.org';
```

For the record we don't support `.xls`, only `.xlsx`.
